tests/integration/jit/function/test_computations.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Transaction event created and no errors despite creating spans
+for a HUGE number of calls.
+*/
+
+/*SKIPIF
+<?php
+
+require('skipif.inc');
+
+*/
+
+/*INI
+newrelic.distributed_tracing_enabled=1
+newrelic.transaction_tracer.threshold = 0
+error_reporting = E_ALL
+opcache.enable=1
+opcache.enable_cli=1
+opcache.file_update_protection=0
+opcache.jit_buffer_size=32M
+opcache.jit=function
```

Comment:
The default appears to be 'tracing/on' instead of 'function'.  Is there a reason we change this from what might be the most common value for our users?

Reply:
Ok I think get it there is a 'function' and 'tracing' directory for testing each mode?